lib/nifty_services/base_crud_service.rb

```diff
@@ -67,7 +67,7 @@ def record_allowed_attributes
     def array_values_from_hash(options, key, root = nil)
       options = options.symbolize_keys
 
-      if root.present?
+      if !!root
```

Comment:
`options = (options[root.to_sym] || {}).symbolize_keys if root` is better too, and, I'm not sure if this method is used in this codebase, maybe we can remove it from this class, or at least document in README as a helper method